scopes/ui-foundation/no-components/no-components.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ExternalLink } from '@teambit/ui.external-link';
+import styles from './no-components.module.scss';
+
+export type NoComponentsProps = {
+  name: string;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+/**
+ * page to be shown when workspace/scope has no components
+ */
+export function NoComponents({ name, children }: NoComponentsProps) {
```

Comment:
I don't like the name. How about - `EmptyComponentsGallery` ?

Reply:
true. will update